packages/samples/mvp-dapp/src/views/read.view.ts

```diff
@@ -0,0 +1,13 @@
+// import type { ItemData } from '../record/record.model'
+
```

Comment:
```suggestion
```

Reply:
Will be uncommented after model branch merged.

Reply:
Gotcha